test/functional/test-amp-context.js

```diff
@@ -61,6 +61,37 @@ describe('3p ampcontext.js', () => {
     windowMessageHandler = undefined;
   });
 
+  it('should call addEventListener with correct eventType', () => {
+    win.name = generateSerializedAttributes();
+    const context = new AmpContext(win);
+    expect(context).to.be.ok;
+    const addEventListenerSpy = sandbox.spy(win, 'addEventListener');
+    context.reportError();
+
+    expect(addEventListenerSpy).to.have.been.called;
+    expect(addEventListenerSpy)
+        .to.have.been.calledWith('error');
+  });
+
+  it('should send error and message when reportError()', () => {
+    win.name = generateSerializedAttributes();
+    const context = new AmpContext(win);
+    expect(context).to.be.ok;
+
+    // Resetting since a message is sent on construction.
+    windowPostMessageSpy.reset();
+    context.reportError();
+
+    // window.context should have sent postMessage sending 3p errors
+    expect(windowPostMessageSpy.calledWith({
```

Comment:
nit: This expect check is not functional, use  `expect(windowPostMessageSpy).to.be.calledWith()`